src/main/java/duke/command/AddCommand.java

```diff
@@ -0,0 +1,105 @@
+package duke.command;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+import duke.DukeException;
+import duke.storage.Storage;
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.TaskList;
+import duke.task.Todo;
+import duke.ui.Ui;
+
+
+/**
+ * Represents a command to add tasks to the task list in the Duke application.
+ */
+public class AddCommand extends Command {
+    private String fullCommand;
+    private char type;
```

Comment:
Perhaps this variable name could be clearer, something like taskType? Just to make it very obvious and prevent any ambiguity :-)